crash-handler-process/logger.cpp

```diff
@@ -34,12 +38,27 @@ const std::string getTimeStamp()
 
 void logging_start(std::wstring log_path)
 {
-    if( !log_output_disabled && log_path.size() > 0) 
+    if( log_path.size() == 0 )
+        log_output_disabled = true;
+
+    if( !log_output_disabled ) 
+    {
+        try {
+            fs::path log_file = log_path;
+            fs::path log_file_old = log_file;
+            log_file_old.replace_extension("log.old");
+            
+            fs::remove_all(log_file_old);
+            fs::rename(log_file, log_file_old); 
+        } catch (...) {
+

```

Comment:
Just confirming, is it intended that this catch does nothing?

Reply:
yes. if it fails to remove previous log file there is nothing we can do about it. And it is also pretty unimportant feature so just catch and ignore.

Reply:
Nice. Resolving the conversation then!